UnityProject/Assets/Scripts/Items/Devices/AccessRemote.cs

```diff
@@ -83,17 +82,21 @@ public bool WillInteract(HandApply interaction, NetworkSide side)
 			return Validations.HasComponent<DoorMasterController>(interaction.TargetObject) && Validations.CanApply(interaction.PerformerPlayerScript, interaction.TargetObject, side, false, ReachRange.Unlimited);
 		}
 
+		//FIXME: this shouldn't make its own check and use doors like this. Refactor doors to allow interacting with them with devices instead
+		// and let the door handle the interaction.
```

Comment:
might be worth adding note to make them client predicted?

Reply:
Uhmm, I was going to do the mentioned refactor. Don't know how to do them client predicted

Reply:
fair enough just putting a to-domaybe, is fine